go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/kami-zh/go-capturer v0.0.0-20171211120116-e492ea43421d
 	github.com/pingcap/br v5.0.0-rc.0.20201223100334-c344d1edf20c+incompatible // indirect
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
-	github.com/pingcap/dumpling v0.0.0-20201224084933-34903c7475cf
+	github.com/pingcap/dumpling v0.0.0-20201230072552-5f1acb878f3e
```

Comment:
dumpling has just updated its log print system. You can update to the newest dumpling master branch.

Reply:
https://github.com/pingcap/dm/pull/1310/commits/5f55bf3ef36fef66b02fc9e1db2ad45ae62db840